hypothesis-python/src/hypothesis/internal/charmap.py

```diff
@@ -79,12 +79,20 @@ def charmap():
                 os.close(fd)
                 # Explicitly set the mtime to get reproducible output
```

Comment:
(It’s not part of this patch, but I have no idea what this comment is referring to.)

Reply:
https://docs.python.org/3/library/gzip.html#gzip.GzipFile

It ensures that the resulting file is byte-identical when regenerated, which helps the paranoid :smile:

Reply:
🤦‍♀️ 

How did I miss us setting the `mtime` on the next line.